nemo/collections/common/tokenizers/tokenizer_utils.py

```diff
@@ -0,0 +1,121 @@
+# =============================================================================
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import os
+
+from transformers import AlbertTokenizer, BertTokenizer, RobertaTokenizer
+
+import nemo
+from nemo.utils import logging
+
+try:
+    __megatron_utils_satisfied = True
+    from nemo.collections.nlp.nm.trainables.common.megatron.megatron_utils import (
```

Comment:
there is no nemo.collections.nlp.nm.trainables.common.megatron.megatron_utils .... 
Could you please remove this for now or is it actually needed?